viz.yaml

```diff
@@ -77,18 +77,23 @@ publish:
       header: "header_section"
       footer: "footer_section"
       hashes: "url_hashes"
+      projection: "custom_projection"
       map_utils: "map_utils"
       map: "build_map"
       timeslider: "timeslider"
       conus_map_geojson: "conus_map_geojson"
     context:
       resources: ["lib_d3_js", "header_css", "footer_css",
-                  "map_css", "hashes", "map_utils", "timeslider"]
+                  "map_css", "hashes", "projection", "map_utils", "timeslider"]
       sections: ["map"] # adding header and footer here, doubled each on the page
   -
     id: url_hashes
     location: "js/url_hashes.js"
     mimetype: application/javascript
+  -
+    id: custom_projection
+    location: "js/custom_projection.js"
+    mimetype: application/javascript
```

Comment:
Can we add a comment right here about how this needs to come before `build_map`? Then, it's recorded in the actual code where someone might change it, and not just in your PR description.